Mlem/Views/Tabs/Settings/Components/Views/Appearance/AppearanceSettingsView.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  Appearance.swift
+//  Mlem
+//
+//  Created by David Bureš on 08.05.2023.
+//
+
+import SwiftUI
+
+struct AppearanceSettingsView: View {
+
+    @AppStorage("lightOrDarkMode") var lightOrDarkMode: UIUserInterfaceStyle = .unspecified
+    
+    var body: some View {
+        VStack {
+            List {
+                Section {
+                    NavigationLink {
+                        ThemeSettingsView()
+                    } label: {
+                        switch lightOrDarkMode {
+                        case .unspecified:
+                            ThemeLabel(title: "Theme", color1: .white, color2: .black)
+                        case .light:
+                            ThemeLabel(title: "Theme", color1: .white)
+                        case .dark:
+                            ThemeLabel(title: "Theme", color1: .black)
+                        default:
+                            ThemeLabel(title: "Theme", color1: .clear)
+                        }
+                    }
+                    #if !os(macOS) && !targetEnvironment(macCatalyst)
+                    NavigationLink {
+                        IconSettingsView()
+                    } label: {
+                        Label {
+                           Text("App Icon")
+                        } icon: {
+                            IconSettingsView.getCurrentIcon()
+                                .resizable()
+                                .scaledToFit()
+                                .frame(width: 28, height: 28)
+                                .cornerRadius(7)
```

Comment:
For consistency with the rest of the app, this should either be `AppConstants.smallItemCornerRadius` or `AppConstants.largeItemCornerRadius`--let's try it with small, if that looks bad/too blocky we can go with large

Reply:
This should be its own constant imo - smallItemCornerRadius is used for the Headline/Compact post thumbnails, which doesn't relate to the settings labels.

Reply:
I've added a new `AppConstants.settingsIconCornerRadius` value. What do you think?